bitnami/node/templates/deployment.yaml

```diff
@@ -29,8 +29,8 @@ spec:
       {{- end }}
       initContainers:
       - name: git-clone-repository
-        image: "{{ .Values.gitImage.registry }}/{{ .Values.gitImage.repository }}:{{ .Values.gitImage.tag }}"
-        imagePullPolicy: {{ .Values.gitImage.pullPolicy | quote }}
+        image: "{{ template "git.image" . }}"
+        imagePullPolicy: {{ .Values.git.pullPolicy | quote }}
         command: [ '/bin/sh', '-c' , 'git clone {{ .Values.repository }} /app && cd /app && git checkout {{ .Values.revision }}']
```

Comment:
For the new initContainer we should replace this command to allow using two commands in the same container, new Dockerfile is something like:
```
ENTRYPOINT ["git"]
CMD ["--help"]
```